src/udev/udev-worker.c

```diff
@@ -248,7 +248,7 @@ void udev_broadcast_result(sd_device_monitor *monitor, sd_device *dev, EventResu
                                 (void) device_add_property(dev, "UDEV_WORKER_ERRNO_NAME", str);
                         break;
                 }
-                case EVENT_RESULT_EXIT_STATUS_BASE ... EVENT_RESULT_EXIT_STATUS_MAX:
+                case EVENT_RESULT_EXIT_STATUS_BASE_START ... EVENT_RESULT_EXIT_STATUS_MAX:
```

Comment:
If you really want to do this, please add `case EVENT_RESULT_EXIT_STATUS_BASE: assert_not_reached()`

Reply:
Added assert not reached check for EVENT_RESULT_EXIT_STATUS_BASE